squid-amplitude.yaml

```diff
@@ -1,6 +1,6 @@
 manifestVersion: subsquid.io/v0.1
 name: amplitude-squid
-version: 15
```

Comment:
There is a conflict with the version bumps from [this PR](https://github.com/pendulum-chain/pendulum-squids/pull/51/files). However, only the Amplitude Squid version 16 has been deployed so far. We probably need to update the version definitions here once all squids from PR #51 have been deployed.